config/locales/mfa/en.yml

```diff
@@ -7,6 +7,13 @@ en:
     info: Add another layer of security by selecting a multi-factor authentication
       method. We recommend you select at least (2) two different options in case
       you lose one of your methods.
+    non_restricted:
+      button: Add an authentication method
+      heading: Add another authentication method.
+      info_html: You opted to receive a code by SMS/text or a phone call to sign in
+        after you entered your email and password. You are now required to add
+        another authentication method as a backup in case you lose access to
+        your phone. <a href='%{link}'>Learn more</a>
```

Comment:
Not a blocker, but I wonder if we could consider a more descriptive link text to improve accessibility?

Related: https://www.nngroup.com/articles/learn-more-links/

Reply:
I'm referring to the content. Depending whether you agree with my concern, could be worth surfacing with your team.

Reply:
I think if the link was by itself yes, but with the info provided prior to the link I think it follows the aspect of https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html#dfn-programmatically-determined-link-context right?

Reply:
But I think if we want to go over that content decision as a concept we can discuss it in a separate ticket addressing all instances in which we use this pattern.

Reply:
I agree with both of you! Thanks for this discussion. On the one hand, as @aduth said we should do a better job discussing these links when creating content and I'll certainly flag it in our discussion. Especially since we can and should do better.
I also agree with @mdiarra3 in this case, as supported by the article shared by @aduth that it is pretty clear in this particular case, in my opinion: https://www.nngroup.com/articles/learn-more-links/#:~:text=Let%20us%20first%20acknowledge%20that%20if%20the%20text%20that%20precedes%20the%20link%20makes%20it%20obvious%20what%E2%80%99s%20to%20come%2C%20then%20a%20Learn%20More%20label%20isn%E2%80%99t%20devastating%20to%20the%20user%20experience.